be/src/exec/vectorized/file_scan_node.cpp

```diff
@@ -1,336 +0,0 @@
-// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
Do not delete the previous implementation of FileScanNode directly, we need to still generate the old execution plan in the new version to ensure that the feature can be rollback.